src/js/components/Meter/Bar.js

```diff
@@ -5,10 +5,19 @@ import { defaultProps } from '../../default-props';
 import { parseMetricToNum } from '../../utils';
 
 import { StyledMeter } from './StyledMeter';
-import { strokeProps, defaultColor } from './utils';
+import { strokeProps, defaultColor, getPathByDirection } from './utils';
 
 const Bar = forwardRef((props, ref) => {
-  const { background, max, round, size, thickness, values, ...rest } = props;
+  const {
+    background,
+    max,
+    round,
+    size,
+    thickness,
+    values,
+    direction,
```

Comment:
Using `direction` for 'rtl' wouldn't be consistent with the current Box `direction` prop.
I'm thinking we should leverage the existing `margin` prop with the value `start` or `end` for the RTL behavior, consistently with how it is done for other components with rtl functionality.
Rleated to #3629

Reply:
* I have changed the prop controlling the meter direction from "direction" to "margin" where if the meter has a margin.start  (for example <meter margin={{start:"xsmall"}} ... />) then it behaves as an rtl element.
* Now there's no need to redefine the types as the margin is already defined there.
* the calendar test is failing - although i didn't touch that area in my PR.

Reply:
@ShimiSun some UI tests are failing, i think because its yet to be reviewed by someone. I haven't used chromatic before, so It is a bit unclear for me.

Reply:
> @ShimiSun some UI tests are failing, i think because its yet to be reviewed by someone. I haven't used chromatic before, so It is a bit unclear for me.

@Net301 - You are correct that the Chromatic snapshot just needs to be reviewed since it is a newly added Storybook story. So, looking good.

Also, per:
> Thanks for this PR!
> Once the PR direction is approved, the doc.js and index.d.ts should be completed as well.

It looks like next steps are for the direction to be approved and then some documentation and type definitions.

Reply:
@halocline
- I have used the existing types where the margin.start type already exists in index.d.ts . so I don't think the types need extra modification.
- Regarding the documentaiton: As I understand from the code I should update the DocumentedMeter.propTypes object, override the margin child and explain that if a start is added it transforms the bar meter to "rtl". my questions here is: how do I view the documentation in order to manually test it before pushing?
- Regarding the chromatic review, how is the process done ?  by the core team ?

Reply:
Hi @Net301 -

Regarding documentation, yes in `Meter/doc.js` modify the margin property. To test, you can run `yarn generate-readme` which is a script that generates all of the README markdown files for each component. After running, you will see that `Meter/README.md` will have been modified and can be manually inspected there. As FYI, the `generate-readme` script is also run when `yarn test-update` script is executed.

Regarding Chromatic review, yes, someone from the core team will review.